cdap-docs/included-applications/source/etl/plugins/batchsinks/tpfsparquet.rst

```diff
@@ -1,47 +1,51 @@
 .. meta::
-    :author: Cask Data, Inc.
-    :copyright: Copyright © 2015 Cask Data, Inc.
+    :author: cask data, inc.
+    :copyright: copyright © 2015 cask data, inc.
+
+.. _included-apps-etl-plugins-batch-sinks-tpfsparquet:
 
 ===============================
-Sinks: Batch: TPFSParquet
+batch sinks: tpfsparquet
 ===============================
 
-Sink for a TimePartitionedFileSet that writes data in Parquet format.
-Every time the pipeline runs, a new partition in the TimePartitionedFileSet
+.. rubric:: description
+
+sink for a timepartitionedfileset that writes data in parquet format.
+every time the pipeline runs, a new partition in the timepartitionedfileset
 will be created based on the logical start time of the run.
-All data for the run will be written to that partition.
+all data for the run will be written to that partition.
 
-.. rubric:: Use Case
+.. rubric:: use case
 
-This sink is used whenever you want to write to a TimePartitionedFileSet in Parquet format.
-For example, you might want to create daily snapshots of a database table by reading
+this sink is used whenever you want to write to a timepartitionedfileset in parquet format.
+for example, you might want to create daily snapshots of a database table by reading
 the entire contents of the table and writing to this sink.
 
-.. rubric:: Properties
+.. rubric:: properties
 
-**name:** Name of the TimePartitionedFileSet to which records are written.
-If it doesn't exist, it will be created.
+**name:** name of the timepartitionedfileset to which records are written.
+if it doesn't exist, it will be created.
 
-**schema:** The Avro schema of the record being written to the sink as a JSON Object.
+**schema:** the avro schema of the record being written to the sink as a json object.
 
-**basePath:** Base path for the TimePartitionedFileSet. Defaults to the name of the dataset.
+**basepath:** base path for the timepartitionedfileset. defaults to the name of the dataset.
 
-**filePathFormat:** Format for the time partition, as used by SimpleDateFormat.
-Defaults to formatting partitions as 2015-01-01/20-42.142017372000.
+**filepathformat:** format for the time partition, as used by simpledateformat.
+defaults to formatting partitions as 2015-01-01/20-42.142017372000.
 
-**timeZone:** The string ID for the TimeZone to format the date in. Defaults to using UTC.
-This setting is only used if filePathFormat is not null.
+**timezone:** the string id for the timezone to format the date in. defaults to using utc.
+this setting is only used if filepathformat is not null.
 
-.. rubric:: Example
+.. rubric:: example
 
 ::
 
   {
-    "name": "TPFSAvro",
+    "name": "tpfsavro",
     "properties": {
       "name": "users",
-      "filePathFormat": "yyyy-MM-dd/HH-mm",
-      "timeZone": "America/Los_Angeles",
+      "filepathformat": "yyyy-mm-dd/hh-mm",
```

Comment:
don't change these capitalization is important in both the key and the value. MM means month while mm means minute (http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html)